### PR TITLE
add cinder and nova capacity plugins

### PR DESCRIPTION
**Pull request by auhlig**
cinder capacity plugin

### CODE REVIEW

**Comment by auhlig** [comment]
Good to merge despite decreasing coverage @majewsky?

File: pkg/plugins/capacity_cinder.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		return nil, err
+	}
+
+	length := 0
+	for _, element := range availabilityZoneData.AvailabilityZoneInfo {
+		if element.ZoneState.Available == true {
+			length++
+		}
+	}
+
```
Comment (majewsky) [inline pkg/plugins/capacity_cinder.go]: This paragraph is duplicated further down, and should be removed.

**Comment by auhlig** [comment]
Added nova capacity plugin and incorporate comments. Please review again @majewsky.

**Comment by auhlig** [comment]
[This](https://github.com/sapcc/limes/pull/21/commits/8e54604117095aad7a9208cebfa5a9fde228b32a#diff-11088ebd7f942a7e6932045dead5eaeeR99) is how the pagination works .

File: pkg/plugins/capacity_nova.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+	//returns something like
+	//"volumev2": {
+	//	"cores": total_vcpus,
+	//	"instances": min(10000 per Availability Zone, local_gb/max(flavor size)),
+	//	"ram": total_memory_mb,
+	//}
+	return map[string]map[string]uint64{
+		"computev2": {
+			"cores":     uint64(hypervisorData.HypervisorStatistics.Vcpus),
```
Comment (reimannf) [inline pkg/plugins/capacity_nova.go]: Is this the overcommitted value? I can remember that `openstack hypervisor stats show` does not consider the overcommit factor.

Comment (majewsky) [inline pkg/plugins/capacity_nova.go]: I added a configurable overcommit factor to the plugin's configuration to account for this.

**Pull request merged**